_includes/install.html

```diff
@@ -19,10 +21,7 @@
 </p>
 
 <pre>
-<b>$ </b>curl https://nixos.org/nix/install | sh
+<b>$ </b>curl -L https://nixos.org/nix/install | sh
 <b>$ </b>. "$HOME/.nix-profile/etc/profile.d/nix.sh"
-<b>$ </b>nix-env -if https://github.com/cachix/cachix/tarball/master --substituters https://cachix.cachix.org --trusted-public-keys cachix.cachix.org-1:eWNHQldwUO7G2VkjpnjDbWwy4KQ/HNxht7H4SSoMckM=
-<b>$ </b>cachix use dapp
-<b>$ </b>git clone --recursive https://github.com/dapphub/dapptools $HOME/.dapp/dapptools
-<b>$ </b>nix-env -f $HOME/.dapp/dapptools -iA dapp seth solc hevm ethsign
+<b>$ </b>nix-env -iA dapp hevm seth solc -if https://github.com/dapphub/dapptools/tarball/master --substituters https://dapp.cachix.org --trusted-public-keys dapp.cachix.org-1:9GJt9Ja8IQwR7YW/aF0QvCa6OmjGmsKoZIist0dG+Rs=
```

Comment:
I decided to not install/use Cachix here because if people want the manual install, I presume they're more advanced and want to know what happens.